xchange-stream-bitmex/src/main/java/info/bitrich/xchangestream/bitmex/BitmexStreamingService.java

```diff
@@ -194,7 +200,12 @@ protected DefaultHttpHeaders getCustomHeaders() {
   @Override
   protected String getChannelNameFromMessage(JsonNode message) throws IOException {
     String table = message.get("table").asText();
-    if ("order".equals(table) || "funding".equals(table) || "position".equals(table)) {
+    if (table.equals("order")
```

Comment:
would this make more sense as a switch, and if not at least having the constant on the left side of the equality comparison for NPE protection?

Reply:
having a `private static final Set<String> SIMPLE_TABLES = ImmutableSet.of(..)`

and just doing `if (SIMPLE_TABLES.contains(table)) {` would be better

Reply:
I like ImmutableSet.of()